src/iad.cs

```diff
@@ -170,6 +170,8 @@ interface ADInterstitialAdDelegate {
 
 	[Category (allowStaticMembers: true)] // Classic isn't internal so we need this
 	[BaseType (typeof (MPMoviePlayerController))]
+	[Deprecated (PlatformName.iOS, 9,0, message: "Use 'iAdPreroll_AVPlayerViewController' instead.")]
+	[Obsoleted (PlatformName.iOS, 12,0)] // header removed in xcode10 beta5
```

Comment:
You can have messages for obsoleted. Not sure we want to talk about header removal but we could say that we don't expect the API to work since Xcode 10 anymore.

Reply:
Not quite sure what (or how) to say - an ambiguous message probably won't help. It _works_ today and will probably continue to work because of existing applications (will it continue to show ads is another story).

The fact that Apple removed the API is pretty much the "default" meaning for obsoleted and the solution is covered in the deprecated message. Any better ideas ?